### PR TITLE
[DOCS-6608] - Alfresco Governance Services 7.2

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

**Review by tpage-alfresco — APPROVED** [review]
The changes look fine as far as I can tell. I couldn't really review the new files (which have presumably been copy/pasted).

File: governance-services/latest/install/uninstall.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ $ java -jar bin\alfresco-mmt.jar uninstall <ModuleId> <WARFileLocation>
     ```bash
     Module 'alfresco-rm-enterprise-share' installed in 'tomcat\webapps\share.war'
     -    Title:        Alfresco Record Management Share Extension
-    -    Version:      7.1
+    -    Version:      7.2
     -    Install Date: Tue Sept 28 08:50:15 UTC 2021
```
Comment (anxumalo) [inline governance-services/latest/install/uninstall.md:60]: _Future update_: we should consider making lines 42 & 60 more generic or update the date

File: governance-services/latest/install/uninstall.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ $ java -jar bin\alfresco-mmt.jar uninstall <ModuleId> <WARFileLocation>
...
        -    Title:        AGS Repo
-       -    Version:      7.1
+       -    Version:      7.2
        -    Install Date: null
        -    Description:   Alfresco Governance Services Repository Extension
     Module 'alfresco-rm-enterprise-repo' installed in 'webapps/alfresco'
        -    Title:        AGS Enterprise Repo
-       -    Version:      7.1
+       -    Version:      7.2
        -    Install Date: Tue Sept 28 08:50:15 UTC 2021
```
Comment (anxumalo) [inline governance-services/latest/install/uninstall.md:42]: _Future update_: we should consider making lines 42 & 60 more generic or update the date

**Review by anxumalo — APPROVED** [review]
LGTM

There are a couple of related in-line comments we can consider for future revisions.

**Pull request merged**